Sources/carton/Model/Template.swift

```diff
@@ -0,0 +1,185 @@
+          ],
+          dependencies: [
+              \(dependencies.map(\.description).joined(separator: ",\n"))
+          ],
+          targets: [
+              .target(
+                  name: "HelloWorld",
+                  dependencies: [
+                      \(targetDepencencies.map(\.description).joined(separator: ",\n"))
+                  ]),
+              .testTarget(
+                  name: "\(project.name)Tests",
+                  dependencies: ["\(project.name)"]),
+          ]
+      )
+      """
+        .write(to: $0)
+    }
+  }
+}
+
+// MARK: Templates
+extension Templates {
+  struct Basic : Template {
+    static let description: String = "A simple SwiftWasm project."
+    
+    static func create(on fileSystem: FileSystem,
+                       project: Project,
+                       _ terminal: TerminalController) throws {
+      try fileSystem.changeCurrentWorkingDirectory(to: project.path)
+      try createPackage(type: .executable,
+                        fileSystem: fileSystem,
+                        project: project,
+                        terminal)
+      try createManifest(fileSystem: fileSystem,
+                         project: project,
+                         terminal)
+    }
+  }
+}
+
+extension Templates {
+  struct Tokamak : Template {
+    static let description: String = "A simple Tokamak project."
+    
+    static func create(on fileSystem: FileSystem,
+                       project: Project,
+                       _ terminal: TerminalController) throws {
+      try fileSystem.changeCurrentWorkingDirectory(to: project.path)
+      try createPackage(type: .executable,
+                        fileSystem: fileSystem,
+                        project: project,
+                        terminal)
+      try createManifest(fileSystem: fileSystem,
+                         project: project,
+                         dependencies: [
+                          .init(name: "Tokamak", url: "https://github.com/swiftwasm/Tokamak", version: .branch("main"))
+                         ],
+                         targetDepencencies: [
+                          .init(name: "TokamakDOM", package: "Tokamak")
+                         ],
+                         terminal)
+      try fileSystem.writeFileContents(project.path.appending(components: "Sources", project.name, "main.swift")) {
+        """
+        import TokamakDOM
+        import JavaScriptKit
+
+        let document = JSObjectRef.global.document.object!
+        let body = document.body.object!
+        body.style = "margin: 0;"
+
+        let div = document.createElement!("div").object!
+        let renderer = DOMRenderer(ContentView(), div)
+        _ = body.appendChild!(div)
+        """
+          .write(to: $0)
+      }
+      try fileSystem.writeFileContents(project.path.appending(components: "Sources", project.name, "ContentView.swift")) {
+        """
+        import TokamakDOM
+
+        struct ContentView : View {
```

Comment:
**nitpick:** I'm probably fine with the default 4 spaces indentation for new projects, but not sure if spaces before colons is consistent with the colon formatting in say `var body: some View`. Up to you, this is not blocking at all 🙂

```suggestion
        struct ContentView: View {
```